packages/conf-brotli/conf-brotli.0.0.1/opam

```diff
@@ -0,0 +1,13 @@
+opam-version: "1.2"
+maintainer: "fxfactorial"
+authors: "Edgar Aroutiounian"
```

Comment:
Ah, I wasn't clear who the license or authors actually referred to, me or the creators of the original package